### DISCUSSION TITLE
Access denied when importing an image

### DISCUSSION THREAD

**Discussion by supahmustah** [remix-run/remix#2160] [Q&A] [answered]
Hello

I recently run into a little problem I have not had before and I can't find the solution somewhere.
Whenever I try to import an image I get this message:
![image](https://user-images.githubusercontent.com/40064576/156006772-af9cb073-398e-46f5-ac63-0914254a039f.png)
All I do is import it like this:
![image](https://user-images.githubusercontent.com/40064576/156006881-495c937a-0eca-4e98-9ae9-ce057ed9c998.png)

It used to work before, it just suddenly stopped working.
Can someone help me? :)

**Comment by kiliman** [comment, marked as answer]
Where is the image located? 

If it's in `/public/images`, then use `<img src="/images/mountainBg.jpg">`

If it's in `/app/images`, then `import background from "~/images/mountainBg.jpg"` and then you can `<img src={background}>`

The `~` means relative to app folder. You should not `import` files outside of app.

**Reply by supahmustah** [reply]
![image](https://user-images.githubusercontent.com/40064576/156566951-1abba3f2-1f20-479d-a458-a57e8646cce7.png)
image is located inside `app/images`.
Tried both answers, still getting access denied or 404 not found

**Reply by kiliman** [reply]
Can you create a new project and try importing an image? It should work, so perhaps it has something to do with your current project.
